support-internationalisation/build.sbt

```diff
@@ -2,7 +2,7 @@ import sbtrelease.ReleaseStateTransformations._
 
 name := "support-internationalisation"
 
-scalaVersion := "3.3.3"
+scalaVersion := "2.13.13"
```

Comment:
tests that depended on this module for some reason missed it off the classpath when running in intellij, until I changed this default version. (`sbt test` etc was fine)
Once more is updated to scala 3, we might have the same problem with other way around - but let's cross that when we get there.